src/Layers/xrRenderPC_GL/r2_R_sun.cpp

```diff
@@ -345,7 +345,7 @@ struct DumbClipper
         Fvector3 D;
         for (int it = 0; it < int(planes.size()); it++)
         {
-            Fvector4 P = planes [it];
+            const Fvector4 P = planes [it];
```

Comment:
Do you really want to copy the vector? Or maybe make a reference instead of copying?